content/zh/docs/releases/feature-stages/index.md

```diff
@@ -18,52 +18,53 @@ Note: this contains feature status from
-|可靠性 |用户不应期望可靠的功能可能未经测试。                                                                                                                                                                                                                                                                                                                                                                                                     | 由于该功能相对较新，并且可能缺乏完整的端到端测试，因此通过标志启用该功能可能会暴露导致 Istio 不稳定的错误（例如，控制循环中的错误可能会迅速创建过多的对象，耗尽 API 存储空间） . |由于该功能具有 e2e 测试，因此启用该功能不应在不相关的功能中产生新的错误。由于该功能相对较新，因此可能存在小错误。                                                                                                                                                                                                                                                                                                                     | 高的。该功能经过充分测试，适用于所有用途，稳定可靠。 |
-|支持 | Istio 社区没有承诺改进、维护或完成该功能，并且该功能可能会在新版本中完全删除。 | Istio 没有承诺完成该功能 该功能可能会在以后的软件版本中完全删除。| Istio 承诺在后续稳定版本中以某种形式完成该功能，通常这会在 3 个月内发生，但有时会需要更长的时间。发布版本应该同时支持两个连续版本（例如 v1alpha1 和 v1beta1；或 v1beta1 和 v1），至少支持一个发布周期（一般为3个月），让用户有足够的时间升级和迁移资源。 |该功能将继续存在于许多后续版本中。 |
-|推荐用例 |在短期开发或测试环境中，旨在征求用户的早期反馈以制定开发工作。                                                                                                                                                                                                                                                                                                                                      | 在短期开发或测试环境中，由于可升级性的复杂性和缺乏长期支持。 |在开发或测试环境中。在生产环境中作为功能评估的一部分以提供反馈。                                                                                                                                                                                                                                                                                                                                                     | 任何。                                                                                                                                        |
-
-## Istio 特性{#Istio-features}
+本页面列出了每个 Istio 特性的相对成熟度和支持级别。请注意，
+这些阶段是针对项目中的各个功能而不是整个项目。以下是对这些内容的高级描述标签。
+
+## 功能定义阶段  {#feature-phase-definitions}
+
+|      | Experimental                                                                                                               | Alpha                                                                                                                    | Beta                                                                                                                                                                    | Stable                                         |                            |                                |                              |                                 |   |   |   |   |
```

Comment:
The preview looks incorrect:

https://deploy-preview-13985--preliminary-istio.netlify.app/latest/zh/docs/releases/feature-stages/

Reply:
Seems the original table is better.

Reply:
Format problem, I am fixing.

Reply:
Fixed, PTAL.

Reply:
cool!